org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/ServerLayoutKind.java

```diff
@@ -13,6 +13,14 @@
 	 * Layout information stored in the model will be overwritten.
 	 */
 	AUTOMATIC, 
+
+	/**
+	 * The server re-layouts the diagram interactively. In ELK, <em>interactive</em> means
+	 * that the order is not calculated but derived from the positions of the nodes. This
+	 * is close to a partial layout.  
+	 * The layout information must be stored in the model and will be overwritten on layout. 
+	 */
+	INTERACTIVE, 
```

Comment:
This value is not used anywhere in the framework. How would an application use it?